src/PCA9685.c

```diff
@@ -70,7 +70,7 @@ int PCA9685_initPWM(int fd, unsigned char addr, unsigned int freq) {
 
   // send a software reset to get defaults, resets all devices on bus
```

Comment:
Please update this comment to reflect the change

Reply:
Drat, one more thing..  The `README.md` will also need to be updated in `PCA9685_initPWM` to state that only the device at `addr` is reset (instead of "all-devices reset").

Reply:
Ooops, I had already fixed that comment locally but hadn't pushed it ;)